### ISSUE TITLE
panic: runtime error: cgo argument has Go pointer to Go pointer

### ISSUE THREAD

**Issue by posto** [rjeczalik/notify#104]
Notify crashes on Go 1.6.3 with the following error message. 
I noticed the already closed issue #90 is related to the same problem.
```
exit status 1
panic: runtime error: cgo argument has Go pointer to Go pointer

goroutine 1 [running]:
panic(0x429d160, 0xc82008cdc0)
        /usr/local/opt/go/libexec/src/runtime/panic.go:481 +0x3e6
github.com/rjeczalik/notify.(*stream).Start(0xc8200a29c0, 0x0, 0x0)
        /Users/x/code/golang/src/github.com/rjeczalik/notify/watcher_fsevents_cgo.go:131 +0x270
github.com/rjeczalik/notify.(*fsevents).watch(0xc82000a3c0, 0xc8200a2840, 0x35, 0x100001b00, 0x0, 0x0)
        /Users/x/code/golang/src/github.com/rjeczalik/notify/watcher_fsevents.go:206 +0x313
github.com/rjeczalik/notify.(*fsevents).RecursiveWatch(0xc82000a3c0, 0xc8200a2840, 0x35, 0x1b0000001b00, 0x0, 0x0)
        /Users/x/code/golang/src/github.com/rjeczalik/notify/watcher_fsevents.go:258 +0x51
go.(*struct { github.com/rjeczalik/notify.watcher; github.com/rjeczalik/notify.recursiveWatcher }).RecursiveWatch(0xc820010360, 0xc8200a2840, 0x35, 0xc800001b00, 0x0, 0x0)
        <autogenerated>:50 +0x7c
github.com/rjeczalik/notify.(*recursiveTree).Watch(0xc820014230, 0xc8200a2840, 0x35, 0xc8200a66c0, 0xc82008cd58, 0x1, 0x1, 0x0, 0x0)
        /Users/x/code/golang/src/github.com/rjeczalik/notify/tree_recursive.go:286 +0xafc
github.com/rjeczalik/notify.Watch(0xc82008cd70, 0xa, 0xc8200a66c0, 0xc82008cd58, 0x1, 0x1, 0x0, 0x0)
        /Users/x/code/golang/src/github.com/rjeczalik/notify/notify.go:64 +0x7d
github.com/cortesi/modd.Watch(0xc82008cd40, 0x1, 0x1, 0x5f5e100, 0xc8200a6660, 0x0, 0x0)
        /Users/x/code/golang/src/github.com/cortesi/modd/modd.go:285 +0x278
main.main()
        /Users/x/code/golang/src/github.com/cortesi/modd/cmd/modd/main.go:99 +0x1e7a
```

What I already tried is to `go get -u` both notify and modd, but it still crashes.

**Comment by rjeczalik** [comment]
Apparently the modd binary is not rebuilt, modd has notify vendored in old version or the process is not restarted.

The stacktrace does not point to a valid line:

```
/Users/x/code/golang/src/github.com/rjeczalik/notify/watcher_fsevents_cgo.go:131
```

https://github.com/rjeczalik/notify/blob/master/watcher_fsevents_cgo.go#L131

**Comment by rjeczalik** [comment]
The issue you linked was fixed by #93. If you attach new stacktrace that refers to HEAD of master of notify project it means the problem is still there and I'd take a look. For now closing.
